zap/src/main/java/org/zaproxy/zap/spider/parser/SpiderParser.java

```diff
@@ -76,6 +78,18 @@ public void removeSpiderParserListener(SpiderParserListener listener) {
         this.listeners.remove(listener);
     }
 
+    /**
+     * Notify the listeners that a resource was found.
+     *
+     * @param resourceFound the http message containing the response.
+     * @since 2.11.0
+     */
+    protected void notifyListenersResourceFound(SpiderResourceFound resourceFound) {
```

Comment:
This should be the only method being added, older methods should be deprecated.

Reply:
Resolved via last commit, previous notify... methods are now deprecated